test/views/PaymentsIntoPensionsStatusTestSupport.scala

```diff
@@ -0,0 +1,167 @@
+/*
+ * Copyright 2022 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package views
+
+import controllers.pensions.paymentsIntoPension.PaymentsIntoPensionFormProvider
+import controllers.pensions.paymentsIntoPension.routes.PaymentsIntoPensionsStatusController
+import forms.YesNoForm
+import models.AuthorisationRequest
+import org.jsoup.Jsoup
+import org.jsoup.nodes.Document
+import play.api.i18n.Messages
+import play.api.mvc.AnyContent
+import support.ViewUnitTest
+import views.html.pensions.paymentsIntoPensions.PaymentsIntoPensionsStatusView
+import models.pension.pages.PaymentsIntoPensionsStatusPage
+
+class PaymentsIntoPensionsStatusTestSupport extends ViewUnitTest {
+
+  object Selectors {
+    val paragraphTextSelector: String = "p.govuk-body"
```

Comment:
make sure to remove paragraph refs from these tests as it does not exist

Reply:
Done 👍